aries-site/src/data/wcag/components.json

```diff
@@ -12,6 +12,24 @@
       { "rule": "2.4.9", "status": "conditional" },
       { "rule": "3.2.1", "status": "passed" },
       { "rule": "4.1.2", "status": "passed" }
+    ],
+    "menu": [
+      { "rule": "1.4.11", "status": "passed" },
+      { "rule": "1.4.13", "status": "passed" },
+      { "rule": "1.4.3", "status": "passed" },
+      { "rule": "2.1.1", "status": "passed" },
+      { "rule": "2.1.2", "status": "passed" },
+      { "rule": "2.4.7", "status": "passed" },
+      { "rule": "2.4.11", "status": "passed" },
+      { "rule": "2.4.12", "status": "passed" },
+      { "rule": "2.4.13", "status": "passed" },
+      { "rule": "2.4.4", "status": "conditional" },
+      { "rule": "2.4.5", "status": "passed" },
```

Comment:
I think 2.4.5 isn't directly related to Menu is a more broad site architecture rule

Reply:
thats fair Ill remove